components/atoms/TokenIcon.tsx

```diff
@@ -4,26 +4,29 @@ import { Image } from "./Image";
 import {aaveAddress} from "../../constants/tokens";
 import { dictionaryTokenIcon } from "../../constants/tokenIcon";
 import { CHAIN_IDS } from "../../constants/chains";
+import { useProtocol } from "../../constants/protocol";
+
 
-// in 
 interface TokenIconInterface {
     address: string // the address of the asset
     size: number // the size of the icon asset
     chainId: number
```

Comment:
given you removed it from the component's props, let's remove it from the interface